crt_portal/cts_forms/question_text.py

```diff
@@ -0,0 +1,52 @@
+from django.utils.translation import gettext_lazy as _
+
+# contact
+CONTACT_QUESTIONS = {
+    'contact_title': _('Contact information'),
+    'contact_name_title': _('Your name'),
+    'contact_first_name': _('First name'),
+    'contact_last_name': _('Last name'),
+    'contact_info_tite': _('Contact information'),
```

Comment:
Is this a maybe a duplicate of `contact_title`? Not seeing the intended key, `contact_info_title`, referenced anywhere

Reply:
Ah I see it in use now in `report_data.html` but looks like we get the same effect by just leveraging `contact_title`.